doc/man1/openssl.pod

```diff
@@ -556,6 +556,18 @@ Read the password from standard input.
 
 =head1 ENVIRONMENT
 
+The OpenSSL library can be take some configuration parameters from the
+environment.  Some of these variables are listed below.  For information
+about specific commands, see L<openssl-engine(1)>, L<openssl-provider(1)>,
+L<openssl-rehash(1)>, and L<tsget(1)>.
+
+For information about specifying a configuration file, and the use of
+environment variables within that file, see
+L<OPENSSL_config(3)> and L<config(5)>.
```

Comment:
This link is sufficient (and more complete): `L<config(5)/ENVIRONMENT>`

Reply:
Also, `within that file` is incorrect.  The environment variables mentioned in config.pod are universal within libcrypto.

Reply:
I rewrote it:
```
For information about specifying a configuration file add L<OPENSSL_config(3)>.
For information about the use of environment variables in configuration,
see L<config(5)/ENVIRONMENT>.
```

Reply:
The first sentence makes no sense.
And I repeat, those environment are not used *in* configuration...  or well, that depends what you mean with "in configuration", but see it like this, `OPENSSL_CONF` is used to find the config file, `OPENSSL_ENGINES` is the directory to load an engine from (so for example, `ENGINE_by_id()` uses that variable), and `OPENSSL_MODULES` is the directory where other modules (such as provider modules) can be found (so for example, `OSSL_PROVIDER_load()` will use it to find the FIPS provider module).  The only variable that's literally used *in* configuration is `OPENSSL_CONF_INCLUDE`.

It's quite possible that we should document the environment variables we use internally in a separate page.  Another man7 page, perhaps?  I'm willing to do that.

Reply:
Well, it's an English thing I think.  How about this: "For information about how the configuration file can use environment variables, see L<config(5)/ENVIRONMENT>."

I removed the first sentence.

I think an additional man7 page on those environment variables.  I added one to this PR and pushed a new commit.